### ISSUE TITLE
Flask-RestX v1.0.5 release & breathing some new life into this project!

### ISSUE THREAD

**Issue by peter-doggart** [python-restx/flask-restx#510]
Afternoon `flask-restx` community! 

With thanks to @jdieter and @ziirish all the compatibility warnings with `flask` 2.3 should be gone in `flask-restx` v1.0.5 which I have just released (hopefully :crossed_fingers:)! If not, please open an issue and let me know! 

Aside from keeping up to date with upstream changes in `flask` and general maintenance, I would like to breathe some new life into this project. 

With that in mind, I would like to ask you, the community, who use this project, what your top priorities for bug fixes and new features or changes are. Big or small, things that don't work or things that you just don't like. I can't promise I will have time to fix or implement all of them, but I do promise to read every suggestion and engage with you. I will also make an effort to review and integrate PRs, so if anyone wants to become a contributor, I would encourage you to do so! 

Anything related to model redesign can go over on #59, but for anything else, please comment below.

**Comment by beautah** [comment]
Just couldn't help myself and had to drop a note to thank all the contributors for this release, I just upgraded (from something like 0.5) and everything is faster with zero pain - zero breaking changes and running a modern flask version. Kudos, so much kudos. I have several projects I was dreading migrating from flask-restplus/restx.

**Comment by davidjb99** [comment]
Thank you for taking over this project @peter-doggart this is a great framework and I’ve enjoyed using it for some time. 

I think better support for OpenAPI generation would be most welcome.

**Comment by peter-doggart** [comment]
@davidjb99 Thanks for your feedback. In terms of better support for OpenAPI generation, what would you like to see going forwards? Obviously, support for OpenAPI v3 has been suggested for a long time, but is there anything else in terms of customization or support you think would be useful?

**Comment by PanderMusubi** [comment]
According to https://pypi.org/project/flask-restx/#history there is also a version 1.1.0 However, that is not yet published here on GitHub.

**Comment by peter-doggart** [comment]
@PanderMusubi Fixed that now. I must have forgotten to do it. It's the only part of the release pipeline that isn't automated by github actions currently.

**Comment by cyrlop** [comment]
Thanks for the great work!
> I would like to ask you, the community, who use this project, what your top priorities for bug fixes and new features

For me the top priority is #26 :smiley: (is support for v2 needed?)

**Comment by thedumbterminal** [comment]
Hi, great news!
I've been waiting a while to get the following PR reviewed: https://github.com/python-restx/flask-restx/pull/442 which resolves this issue: https://github.com/python-restx/flask-restx/issues/438

Let me know if I need to do any changes.

Thanks in advance

**Comment by rlleshi** [comment]
Hello @peter-doggart. I haven't noticed so much progress so far so I was wondering if you are still planning to maintain this project.

**Comment by peter-doggart** [comment]
@rlleshi Yes, but mostly only for maintenance (keeping up with flask releases etc) currently due to time constraints and the fact that a lot of the project is stable. 

Reviewing outstanding PRs is still on my to-do list. The issue is there are so many to get through and I want to make sure we don't accidentally break anything in the process! 

OpenAPI v3 support and other major changes that go deep into flask-restx are unlikely in the short term though.

**Comment by abompard** [comment]
> OpenAPI v3 support and other major changes that go deep into flask-restx are unlikely in the short term though.

No hard commitment, but if I were to send a PR to add OpenAPI v3 support to flask-restx, would you review and consider it @peter-doggart ? Or would you rather see it done by someone from the "core team" or yourself? (which I can totally understand, it's more work to review a newcomer's PR)

**Comment by terryluan12** [comment]
> > OpenAPI v3 support and other major changes that go deep into flask-restx are unlikely in the short term though.
> 
> No hard commitment, but if I were to send a PR to add OpenAPI v3 support to flask-restx, would you review and consider it @peter-doggart ? Or would you rather see it done by someone from the "core team" or yourself? (which I can totally understand, it's more work to review a newcomer's PR)

Just bumping this! Also would definitely be willing to help with this endeavor
